getting-started/minishift_install.adoc

```diff
@@ -0,0 +1,224 @@
+:org: AeroGear
+//source: https://github.com/aerogear/minishift-mobilecore-addon/blob/master/README.md
+
+
+[[setting-up-minishift]]
+= Setting up AeroGear Mobile Services onMinishift
+
+. link:https://docs.openshift.org/latest/minishift/getting-started/installing.html[Install Minishift]
++
+Check Minishift version is at least 1.11.0:
++
+[source,bash]
+----
+$ minishift version
+minishift v1.11.0+4459917
+----
+
+. link:https://docs.openshift.org/latest/cli_reference/get_started_cli.html#installing-the-cli[Install OC]
+
+. Enable Minishift experimental addons:
++
+[source,bash]
+----
+export MINISHIFT_ENABLE_EXPERIMENTAL=y
+----
++
+NOTE: If this variable is not set, you might be see errors similar to `Error: unknown flag: --service-catalog`.
+
+. Clone the link:https://github.com/aerogear/minishift-mobilecore-addon[minishift-mobilecore-addon] repo.
+
+
+. Add minishift-mobilecore to the minishift add-ons catalog:
++
+[source,bash]
+----
+minishift addon install -f minishift-mobilecore-addon/
+minishift addons enable mobilecore
+----
++
+NOTE: On subsequent installs, make sure to update the `minishift-mobilecore-addon` repo (`git pull`) before running these commands to ensure the addon and *mobile-core* versions are compatibile.
+
+. Set the following environment variables, replacing `aerogearcatalog` with your own Dockerhub organization if required:
++
+[source,bash]
+----
+export DOCKERHUB_USERNAME=<docker hub username>
+export DOCKERHUB_PASSWORD=<docker hub password>
+----
++
+NOTE: If you use a Docker Hub organization, set a variable for it using `export DOCKERHUB_ORG=<your docker hub org>`, otherwise the addon will default the value to `aerogearcatalog`.
```

Comment:
This is true, but what really decides whether the default is used, is whether or not this value is present in the addon-env value in the `minishift config set` command.